templates/index.html

```diff
@@ -526,89 +521,82 @@ <h2 class="u-sv3">A selection of customers</h2>
               <td data-heading="Fully managed deployment" class="table-cell "><i class="p-icon--success">Yes</i></td>
             </tr>
             <tr>
-              <th>Vendor support</th>
+              <th>Bug-fix and operational support</th>
               <td data-heading="Appliance" class="table-cell"><span data-heading="No">&ndash;</span></td>
-              <td data-heading="Custom deployment" class="table-cell"><i class="p-icon--success">Yes</i></td>
-              <td data-heading="Fully managed deployment" class="table-cell"><i class="p-icon--success">Yes</i></td>
-            </tr>
-            <tr>
-              <th>Monitoring</th>
-              <td data-heading="Appliance" class="table-cell"><span data-heading="No">&ndash;</span></td>
-              <td data-heading="Custom deployment" class="table-cell"><i class="p-icon--success">Yes</i></td>
+              <td data-heading="Regular deployment" class="table-cell"><i class="p-icon--success">Yes</i></td>
               <td data-heading="Fully managed deployment" class="table-cell"><i class="p-icon--success">Yes</i></td>
             </tr>
             <tr>
               <th>SLAs</th>
               <td data-heading="Appliance" class="table-cell"><span data-heading="No">&ndash;</span></td>
-              <td data-heading="Custom deployment" class="table-cell"><span data-heading="No">&ndash;</span></td>
+              <td data-heading="Regular deployment" class="table-cell"><span data-heading="No">&ndash;</span></td>
               <td data-heading="Fully managed deployment" class="table-cell"><i class="p-icon--success">Yes</i></td>
             </tr>
             <tr class="u-hide--small">
               <th></th>
-              <td data-heading="Appliance" class="table-cell" style="overflow: unset;">
-                <span class="p-contextual-menu--left" style="z-index: 10;">
-                  <button class="p-button--neutral p-contextual-menu__toggle has-icon" aria-controls="menu-3"
-                          aria-expanded="false" aria-haspopup="true">
-                          <span>Deploy now</span><i class="p-icon--chevron-down p-contextual-menu__indicator"></i>
-                      </button>
-                  <span class="p-contextual-menu__dropdown" id="menu-3" aria-hidden="true" >
-                    <span class="p-contextual-menu__group">
-                      <a href="https://anbox-cloud.io/docs/howto/install-appliance/aws" class="p-contextual-menu__link">on AWS</a>
-                      <a href="https://anbox-cloud.io/docs/tut/installing-appliance" class="p-contextual-menu__link">on bare metal</a>
-                    </span>
-                  </span>
-                </span>
+              <td data-heading="Appliance" class="table-cell">
+                <a href="https://anbox-cloud.io/docs/howto/install-appliance/landing" class="p-button--neutral">Install now</a>
```

Comment:
If we can have only one primary button, the appliance one should be it.

Reply:
@morphis Wouldn't it be better to have a "Try" button pointing to the appliance link and an "Install" button pointing to the regular deployment installation? Since the appliance is mostly for prototyping and eventually we want the customers to get to the full version?

Reply:
Actually, we already have the "Try Anbox Cloud" a little below. So for the "Install" button, wouldn't be better to link to the regular deployment?

Reply:
It's a matter of guidance and catching the users eye. Generally I would like the user to always start with the appliance and only go to the regular deployment when he has specific needs. We can drop the topmost "Install" button but if we keep it should go to the "appliance".

Reply:
If that's the case, let's leave it as is for now.